lib/matplotlib/tests/test_backend_qt.py

```diff
@@ -114,6 +114,7 @@ def interrupter():
         print('SUCCESS', flush=True)
 
 
+@pytest.mark.xfail(sys.platform == "darwin", reason='Fails on OSX')
```

Comment:
I don't think this should fail on darwin? It may on CI, but I think we should put that somewhere else if possible... Especially because this will fail me locally.

Running on darwin I get:
```
lib/matplotlib/tests/test_backend_qt.py::test_sigint[show-kwargs0] PASSED            [  5%]
lib/matplotlib/tests/test_backend_qt.py::test_sigint[pause-kwargs1] PASSED           [  7%]
```

Reply:
I have pyqt5, version 5.15.4 installed in this env if that is any help.

Reply:
You can check for `'CI' in os.environ` if you think it's going to be CI-specific.

Reply:
Interesting, I thought I saw these locally as well.  I'll try to check tomorrow when I get home.

Reply:
I think it started failing with qt6. The current CI uses qt5 and there it passes.

Reply:
Now it XPASS:es, but it is not clear which qt-version is used.

```
XPASS lib/matplotlib/tests/test_backend_qt.py::test_sigint[pause-kwargs1] Fails on OSX
XPASS lib/matplotlib/tests/test_backends_interactive.py::test_interactive_thread_safety[{'MPLBACKEND': 'qtagg', 'QT_API': 'PySide2'}] 
XPASS lib/matplotlib/tests/test_backends_interactive.py::test_interactive_thread_safety[{'MPLBACKEND': 'qtcairo', 'QT_API': 'PyQt5'}] 
XPASS lib/matplotlib/tests/test_backends_interactive.py::test_interactive_thread_safety[{'MPLBACKEND': 'qtcairo', 'QT_API': 'PyQt6'}] 
XPASS lib/matplotlib/tests/test_backends_interactive.py::test_interactive_thread_safety[{'MPLBACKEND': 'qtcairo', 'QT_API': 'PySide6'}] 
XPASS lib/matplotlib/tests/test_backends_interactive.py::test_interactive_thread_safety[{'MPLBACKEND': 'qtcairo', 'QT_API': 'PySide2'}] 
```

I'll remove that line in the web interface and see what happens (so please squash merge...).

Reply:
There seems to be some xfail that can be removed in `test_interactive_thread_safety` as well, but I cannot really figure out which...